.editorconfig

```
@@ -0,0 +1,34 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# PHP files
+[*.php]
+indent_style = space
+indent_size = 4
+
+# MD files
+[*.md]
+indent_style = space
+indent_size = 4
```

GitHub-flavored markdown uses two spaces and the end of a line to indicate a linebreak, so `trim_trailing_whitespace` should be disabled for markdown files.

```suggestion
indent_size = 4
trim_trailing_whitespace = false
```